charts/argo-cd/templates/NOTES.txt

```diff
@@ -8,6 +8,14 @@ In order to access the server UI you have the following options:
       - Add the annotation for ssl passthrough: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/ingress.md#option-1-ssl-passthrough
       - Add the `--insecure` flag to `server.extraArgs` in the values file and terminate SSL at your ingress: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/ingress.md#option-2-multiple-ingress-objects-and-hosts
 
+In case you are deploying on openshift 4
+3. You have to use the Openshift Routes. For that use folowing command:
+
+   oc create route passthrough YOUR_ROUTE_NAME --service=YOUR_SERVICE --port=https --insecure-policy=Redirect
```

Comment:
Why hardcoding a dummy name for service? You have access to helm templating here. Please use:
```suggestion
   oc create route passthrough YOUR_ROUTE_NAME --service={{ template "argo-cd.server.fullname" . }} --port=https --insecure-policy=Redirect
```